frontend/store/auctions.ts

```diff
@@ -165,9 +182,26 @@ export const actions = {
             console.error('fetch auction error', error);
             commit('setError', error.message);
         } finally {
+            commit('refreshLastUpdated');
             commit('setAreAuctionsFetching', false);
         }
     },
+    async updateSingleAuction({ commit, rootGetters }: ActionContext<State, State>, auctionId: string) {
+        const network = rootGetters['network/getMakerNetwork'];
+        if (!network) {
+            return;
+        }
+        commit('setIsSelectedAuctionFetching', true);
+        try {
+            const auction = await fetchSingleAuctionById(network, auctionId);
+            commit('setAuction', auction);
+        } catch (error) {
+            console.error('fetch auction error', error);
+            commit('setError', error.message);
```

Comment:
[This comment](https://github.com/sidestream-tech/unified-auctions-ui/pull/251#discussion_r869107949) was not resolved: fetching a single auction will still set error over the whole table in the inactive panel (not even visible on mobile) while the auction page will display nothing. 

The error should be stored either to `selectedAuctionError` hack  or via `auction.error` hack or better via `auctionErrorsStore`. Which is then should be passed to the auction page alert

Reply:
I made a new auctions error store, where we now push errors according to auctionId.